src/components/utils/getNumbers.ts

```diff
@@ -0,0 +1,9 @@
+export const getNumbers = (start: number, end: number) => {
+  const items: number[] = [];
+
+  for (let i = start; i <= end; i++) {
```

Comment:
don't use cycle)))

Reply:
Tell me why regular for-loop is bad here? As for me that looks super clear)